resources/views/dashboard/controllers/events/index.blade.php

```diff
@@ -78,6 +82,27 @@ class="fas fa-info-circle"></i></a>)
                             </td>
                         @endif
                     </tr>
+                    <div class="modal fade" id="denylistEvent" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
Your modal is inside of the foreach loop and it should not be - this is rendering n number of modals with the same ID since it is inside the loop. Furthermore, this is preventing proper execution because the $e->id is the last ID, not the current ID. Move this modal outside of the foreach loop and then pass the $e->id to the modal when the user clicks the delete button.